packages/jest-cli/src/Runtime/Runtime.js

```diff
@@ -363,7 +364,17 @@ class Runtime {
         !this._resolver.getModule(moduleName) &&
         !this._resolver.getMockModule(moduleName)
       ) {
-        absolutePath = this._resolveModule(from, moduleName);
+        if (moduleName) {
+          const virtualMockPath = this._getVirtualMockPath(from, moduleName);
```

Comment:
I have to performance test this. `path.normalize` is pretty slow I think, so it would be good to avoid it in some way.